evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -367,13 +369,13 @@ def test_fit_correct_inputs(mock_ohe_fit_transform, mock_imputer_fit_transform,
     X = pd.DataFrame(X)
     y = pd.Series(y)
     graph = {'Imputer': [Imputer], 'OHE': [OneHotEncoder, 'Imputer.x', 'Imputer.y']}
-    expected_x = ww.DataTable(pd.DataFrame(index=X.index, columns=X.index).fillna(1))
+    expected_x = ww.DataTable(pd.DataFrame(index=X.index, columns=X.columns).fillna(1))
```

Comment:
This isn't a "bug" per se, but the imputer will return one column for each input column, not more which is what this was doing